src/scenario/AbstractDataCollection.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace scenario;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use interfaces\scenario\model\data\DataCollectionInterface;
+
+/**
+ *
+ * @author kevinfrantz
+ *        
+ */
+class AbstractDataCollection extends ArrayCollection implements DataCollectionInterface
```

Comment:
Welchen Mehrwert bietet uns diese Klasse?
Sie implementiert nichts, das Interface ist auch leer.
Abstrakte Klassen sollten besser auch als `abstract` definiert werden.

Das habe ich bei mehreren Klassen/Interfaces festgestellt - kannst du uns deine Idee dazu erläutern?

Reply:
Ehrlich gesagt habe ich nur vergessen die Klassen abstract zu definieren. Ich habe Eclipse statt PHPStorm verwendet und in diesem werden mir manche Bezeichnungsfehler nicht gehighlighted.

Ich denke hier spiegelt sich lediglich das Trauma wieder in Programme mit mangelnden Abstraktionsebenen neue Funktionen zu implementieren. Die Frage nach der Sinnhaftigkeit kann ich nachvollziehen und mit Nonsense beantworten ;)